src/librustc_mir/mir_map.rs

```diff
@@ -180,23 +181,6 @@ impl<'a, 'gcx: 'tcx, 'tcx> MutVisitor<'tcx> for GlobalizeMir<'a, 'gcx> {
 ///////////////////////////////////////////////////////////////////////////
 // BuildMir -- walks a crate, looking for fn items and methods to build MIR from
```

Comment:
It seems this comment refers to the removed BuildMir struct.
Should it also be removed?